docs/source/rules.rst

```diff
@@ -285,6 +286,20 @@ The following table provides an overview of each rule option, with more details
         # throw a KeyError.  Using req_subkeys avoids this.
         return rec['columns']['address'] == '127.0.0.1'
 
+:severity:
+
+  The ``severity`` keyword argument defines the severity level for this rule. The default is ``low``, but this can be set on a per-rule basis.
+
+  Severities, can be user defined but a recommended approach is to use the following severity levels:
+
+==================  ===========
+**Severity level**  **meaning**
+------------------  -----------
+``low``             ``Alert should be looked at when the user has time to investigate``
```

Comment:
I would hesitate to use SLA's to define severity levels here since those will vary wildly across organizations.